R/apache.sedona.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
+
```

Comment:
Is this a file auto-generated by Rstudio or something else. If so, can you please remove it?

Reply:
It is a convenience for R users who want to collaborate on/fork the project. It is never included in the final package (ignored at build time). Usually included in tidyverse projects, e.g. https://github.com/sparklyr/sparklyr/blob/main/sparklyr.Rproj
But I could remove it.